pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -893,17 +893,15 @@ Block.Tabular.Header:
   type: object
   required:
     - label
-    - startColumn
-    - endColumn
+    - columnSpan
   properties:
     label:
       type: string
-    startColumn:
+    columnSpan:
       type: integer
-      minimum: 0
-    endColumn:
-      type: integer
-      minimum: 0
+      minimum: 1
+      default: 1
+      description: the number of columns this header should occupy in a tabular display
```

Comment:
Suggesting using columnSpan instead of start/stop. The one attribute, combined with the order of the headers in the list of headers is enough to figure out positioning. (Also avoids confusion with, say, having a header be the third in the array, but having startColumn=1.